modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,93 @@ If you have installed the {gitops-title} Operator in the default namespace, run
 $ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
 ----
 
+[id="new-features-1-15-0_{context}"]
+== New features
+
+* With this update, {gitops-title} supports the addition of extra volumes and volume mounts to the `ApplicationSet` Controller components. You can configure these additional resources by specifying the `spec.applicationSet.volumes` and `spec.applicationSet.volumeMounts` fields in the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-5739[GITOPS-5739]
++
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-example
+spec:
+  applicationSet:
+    volumes:
+      - name: <volume-name> #<1>
+        emptyDir: {} #<2>
+    volumeMounts:
+      - name: <volume-name> #<3>
+        mountPath: <mount-path> #<4>
+----
+<1> Specifices the name of the `volume` within the Argo CD `ApplicationSet` Controller.
+<2> Specifies the volume type as an `emptyDir`, which creates an empty directory in the pod.
+<3> Specifices the name of the `volumeMounts` within the Argo CD `ApplicationSet` Controller.
+<4> Specifies the path inside the container where the volume will be mounted.
+
+* With this update, the {gitops-shortname} Operator can now gather and export metrics data that is compatible with Prometheus. The data collected by these metrics provide the following important insights into Argo CD instances:
+** The total number of Argo CD instances managed and reconciled.
+** The number of Argo CD instances in various states, such as pending, available, and so on. 
+** The total time for reconciliation loops to complete per instance.
+** The total Resource usage metrics, including CPU, memory, and the number of `go` routines. 
++
+When the {gitops-shortname} Operator is installed in the default `openshift-gitops-operator` namespace, these metrics are automatically available in the {OCP} console. You can access them by navigating to the *Observe* tab and running queries against the monitoring stack. link:https://issues.redhat.com/browse/GITOPS-5674[GITOPS-5674]
+
+* Before this update, users could not configure some replicas for the Argo Rollouts controller by using the `RolloutManager` custom resource (CR). With this update, high availability (HA) support is introduced by adding a new `.spec.ha` field to the `RolloutManager` CR. link:https://issues.redhat.com/browse/GITOPS-5622[GITOPS-5622]
++
+When you enable high availability, the operator automatically sets the number of replicas to 2, the leader election is activated, and default anti-affinity rules are applied to ensure redundancy and reliability.
```

Comment:
can you please add JIRA link `link:https://issues.redhat.com/browse/GITOPS-5511[GITOPS-5511]` from above to this line

Reply:
Fixed.